website/pages/docs/concepts/storage.mdx

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+page_title: Storage
+sidebar_title: Storage
+description: >-
+  Vault relies on external storage to save its information.
+---
+
+# Storage
+
+As described on our [Architecture](/docs/internals/architecture) page, Vault's storage backend is untrusted storage used purely to keep encrypted information.
+
+## Supported Storage Backends
+
+For enterprise customers, HashiCorp provides official support for Consul and Vault's Integrated Storage as storage backends.
+
+Many other options for storage are available with community support - see our [Storage Configuration](/docs/configuration/storage) section for more information.
+
+## Backups & Restores
+
+Due to the highly flexible nature of Vault's potential storage configurations, providing exacting guidance on backing up and restoring Vault is challenging.
+
+When backing up Vault, there are two pieces to consider:
+
+1. Vault's encrypted data in the storage backend
+2. Configuration files and management scripts for running the Vault server
+
+### Vault's Persisted Data
+
+In an ideal world, backups and restores would be performed while Vault is offline. If offline backups are impossible, we have recommended using a storage backend that supports atomic snapshots (such as [Consul](https://www.consul.io/docs/commands/snapshot.html) or [Integrated Storage](https://www.vaultproject.io/docs/commands/operator/raft#snapshot)).
```

Comment:
"have recommended" sounds weird here, why not just "we recommend"?

Reply:
I'm attempting to further communicate that we recommend it with slightly less preference than taking a backup while Vault is offline. It is a bit awkward, I would prefer to keep it however.

Reply:
I think Nick's suggestion here is not to remove this, but to slightly re-word it.